lib/Command/Configure/Check.php

```diff
@@ -61,13 +62,27 @@ protected function configure(): void {
 				InputOption::VALUE_NONE,
 				'Check requirements to use root certificate'
 			);
+		if ($this->pagePreviewAsImage) {
+			$this
+				->addOption('preview',
+					'p',
+					InputOption::VALUE_NONE,
+					'Check requirements to generate image preview'
```

Comment:
Can we use named parameters here ?

Reply:
hum.... Can we do this in another PR as technical debt?

Motivation: We have others commands with options and inside this file have other options. If I change only this point will be strange because will be the unique with named parameters. If I change the other points will be strange because is outside of context of this PR.

Reply:
Yes. Can u create an issue for working on it ?
My problem is to see value `p` and need to go throw `addOption` definition.

Reply:
Issue created